apps/movies/src/movies.controller.ts

```diff
@@ -1,16 +1,29 @@
-import { Controller, Get, Post, Body, Patch, Param, Delete } from '@nestjs/common';
+import { Controller, Get, Post, Body, Param, UseInterceptors, UploadedFile } from '@nestjs/common';
+import { FileInterceptor } from '@nestjs/platform-express';
+import { UploaderService } from '@app/common/uploader/uploader.service';
 import { MoviesService } from './movies.service';
 import { CreateMovieDto } from './dto/create-movie.dto';
 import { plainToInstance } from 'class-transformer';
 import { TrendedMovieResponseDto } from './dto/trended-movie-response.dto';
 
 @Controller('movies')
 export class MoviesController {
-  constructor(private readonly moviesService: MoviesService) {}
+  constructor(private readonly moviesService: MoviesService, private readonly uploaderService: UploaderService) {}
 
   @Post()
-  create(@Body() createMovieDto: CreateMovieDto) {
-    return this.moviesService.create(createMovieDto);
+  @UseInterceptors(FileInterceptor('image'))
+  async create(@Body() createMovieDto: CreateMovieDto, @UploadedFile() image: Express.Multer.File) {
```

Comment:
```suggestion
  async create(@Body() createMovieDto: CreateMovieDto, @UploadedFile() image: Express.Multer.File): Promise<MovieResponseDto[]> {
```

And don't forget to map response from service to DTO. The same as you do for getTrendedMovies. You can try to use decorator TransformPlainToClass instead of plainToInstance method from class-transform lib.